node_account.proto

```diff
@@ -15,6 +15,18 @@
 
 syntax = "proto3";
 
+message ShareInfo {
+    // prev block number before share distribution
+    uint64 block_num = 1;
+    // share which is used for frost
+    float frozen_share = 2;
+    uint64 reward = 3;
+    // timestamp when reward was done
+    uint64 timestamp = 4;
```

Comment:
timestamp when block with this block_num was committed shoud be equal to BlockInfo.timestamp